### PR TITLE
[docs] Align formatting of locators and filters

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

File: docs/modules/plugins/pages/plugin-mobile-app.adoc [hunk 1]
```diff
@@ -53,22 +53,22 @@ a|`tagName(android.widget.TextView)->filter.attribute(text=Hi)` - `android.widge
...
 |Examples
 
-|XPath
+|`XPath`
 |any
 |Search the app XML source using xpath (not recommended, has {locator-strategy}[performance issues])
 |xpath(//android.widget.TextView[@text='Home']) or By.xpath(//XCUIElementTypeStaticText[@value='Home'])
 
-|Accessibility ID
+|`Accessibility ID`
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-mobile-app.adoc:61]: shouldn't it be written in the following way:
```suggestion
|`accessibilityId `
```
?

File: docs/modules/plugins/pages/plugin-mobile-app.adoc [hunk 2]
```diff
@@ -53,22 +53,22 @@ a|`tagName(android.widget.TextView)->filter.attribute(text=Hi)` - `android.widge
...
 |accessibilityId(menu-toggler) or By.accessibilityId(menu-toggler)
 
-|ID
+|`ID`
 |any
 |See {locator-strategy}[ID] selector strategy
 |id(org.vividus:id/menu-toggler) or By.id(menu-toggler)
 
-|iOS Class Chain
+|`iOS Class Chain`
```
Comment (ikalinin1) [inline docs/modules/plugins/pages/plugin-mobile-app.adoc:71]:
```suggestion
|`iosClassChain`
```

File: docs/modules/plugins/pages/plugin-mobile-app.adoc [hunk 3]
```diff
@@ -53,22 +53,22 @@ a|`tagName(android.widget.TextView)->filter.attribute(text=Hi)` - `android.widge
...
 |xpath(//android.widget.TextView[@text='Home']) or By.xpath(//XCUIElementTypeStaticText[@value='Home'])
 
-|Accessibility ID
+|`accessibilityId `
 |any
 |See {locator-strategy}[Accessibility ID] selector strategy
 |accessibilityId(menu-toggler) or By.accessibilityId(menu-toggler)
 
-|ID
+|`ID`
```
Comment (ikalinin1) [inline docs/modules/plugins/pages/plugin-mobile-app.adoc:66]:
```suggestion
|`id`
```

File: docs/modules/plugins/pages/plugin-mobile-app.adoc [hunk 4]
```diff
@@ -53,22 +53,22 @@ a|`tagName(android.widget.TextView)->filter.attribute(text=Hi)` - `android.widge
 |Description
 |Examples
 
-|XPath
+|`XPath`
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-mobile-app.adoc:56]:
```suggestion
|`xPath`
```

File: docs/modules/plugins/pages/plugin-mobile-app.adoc [hunk 5]
```diff
@@ -53,22 +53,22 @@ a|`tagName(android.widget.TextView)->filter.attribute(text=Hi)` - `android.widge
...
 |any
 |Search the app XML source using xpath (not recommended, has {locator-strategy}[performance issues])
 |xpath(//android.widget.TextView[@text='Home']) or By.xpath(//XCUIElementTypeStaticText[@value='Home'])
 
-|Accessibility ID
+|`accessibilityId `
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-mobile-app.adoc:61]:
```suggestion
|`accessibilityId`
```

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by valfirst — APPROVED** [review]

**Pull request merged**